apps/src/sites/studio/pages/courses/index.js

```diff
@@ -22,6 +22,7 @@ function showCourses() {
   const signedOut = coursesData.signedout;
   const modernElementaryCoursesAvailable =
     coursesData.modernelementarycoursesavailable;
+  const showOceans = coursesData.launch_ai;
```

Comment:
passing this `showOceans` prop around everywhere seems  a bit excessive since the tutorial isn't a super secret, but I like the idea of turning all the promo stuff on in one go 🤷‍♀️